x/auth/client/txbuilder/txbuilder.go

```diff
@@ -38,10 +38,10 @@ func NewTxBuilderFromCLI() TxBuilder {
 
 	return TxBuilder{
 		ChainID:       chainID,
-		AccountNumber: viper.GetInt64(client.FlagAccountNumber),
+		AccountNumber: uint64(viper.GetInt64(client.FlagAccountNumber)),
```

Comment:
This could be confusing if the user enters a negative number, maybe we should sanity check the input